src/backend/src/utils/slack.utils.ts

```diff
@@ -37,3 +37,22 @@ export const sendSlackUpcomingDeadlineNotification = async (workPackage: WorkPac
 
   await sendMessage(LEAD_CHANNEL_SLACK_ID, fullMsg);
 };
+
+export const sendSlackRequestedReviewNotification = async (slackId: string, changeRequest: ChangeRequest): Promise<void> => {
+  if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
+
+  const { requestedReviewers } = changeRequest;
+  const slackIds = requestedReviewers.map(async (reviewer) => await getUserSlackId(reviewer.userId));
+
+  Promise.all(slackIds).then(async (slackIds) => {
+    const reviewerStrings = slackIds.map((reviewerSlackId, index) =>
+      buildUserString(requestedReviewers[index], reviewerSlackId)
```

Comment:
yeah this is for a totally unrelated message. if you read the javadoc youll see its for the upcoming deadlines PR